src/components/Article.jsx

```diff
@@ -1 +1,46 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const firstArticle = {
```

Comment:
It is really weird to store data here, import to `App.tsx` and pass with props to `Article` again. Or create separate folder for data or just leave it in `App`

Reply:
agree)